src/MCPServer/lib/archivematicaMCP.py

```diff
@@ -43,60 +42,40 @@
 import time
 import uuid
 
-os.environ['DJANGO_SETTINGS_MODULE'] = 'settings.common'
-
 import django
 django.setup()
 
+from django.conf import settings as django_settings
 from django.db.models import Q
 
 # This project, alphabetical by import source
 import watchDirectory
-import RPCServer
 from utils import log_exceptions
 
 from jobChain import jobChain
 from unitSIP import unitSIP
 from unitDIP import unitDIP
 from unitFile import unitFile
 from unitTransfer import unitTransfer
+from utils import isUUID
+import RPCServer
 
 from django_mysqlpool import auto_close_db
 import databaseFunctions
 from archivematicaFunctions import unicodeToStr
+import dicts
 
 from main.models import Job, SIP, Task, WatchedDirectory
 
-global countOfCreateUnitAndJobChainThreaded
-countOfCreateUnitAndJobChainThreaded = 0
 
-config = ConfigParser.SafeConfigParser()
-config.read("/etc/archivematica/MCPServer/serverConfig.conf")
+countOfCreateUnitAndJobChainThreaded = 0
 
 # time to sleep to allow db to be updated with the new location of a SIP
 dbWaitSleep = 2
 
-
-limitTaskThreads = config.getint('Protocol', "limitTaskThreads")
-limitTaskThreadsSleep = config.getfloat('Protocol', "limitTaskThreadsSleep")
-limitGearmanConnectionsSemaphore = threading.Semaphore(value=config.getint('Protocol', "limitGearmanConnections"))
-reservedAsTaskProcessingThreads = config.getint('Protocol', "reservedAsTaskProcessingThreads")
 stopSignalReceived = False  # Tracks whether a sigkill has been received or not
 
 
-def isUUID(uuid):
```

Comment:
I've moved this to `utils.py` because it's used from other modules in MCPServer, with the idea of avoiding cyclic imports which are kind of annoying.